crates/rocker/Cargo.toml

```diff
@@ -7,9 +7,10 @@ edition = "2018"
 
 [lib]
 name = "rocker"
+path = "src/lib.rs"
 crate-type = ["dylib"]
 
 [dependencies]
 lazy_static = "1.4.0"
-rustler = "0.21.0"
-rocksdb = { git = "https://github.com/Elzor/rust-rocksdb.git", branch = "stable_5.14.2" }
\ No newline at end of file
+rustler = { git = "https://github.com/rusterlium/rustler.git", branch = "master" }
```

Comment:
I'm not convinced that we should use master branch instead vsn.

Reply:
It seems they only got a RC out for now, I think this PR can wait until 0.22 is officially released.

Reply:
I've upgraded rustler to 0.25